### PR TITLE
support for entity names in estonian translation

### PR DESCRIPTION
**Pull request by priiduonu**

### CODE REVIEW

File: custom_components/husqvarna_automower/translations/et.json [hunk 1]
```diff
@@ -75,19 +184,51 @@
...
+                "name": "Kaart",
                 "state": {
                     "cleaning": "Niidab",
                     "docked": "Dokitud",
                     "error": "Viga",
-                    "idle": "[%key:common::state::idle%]",
-                    "off": "[%key:common::state::off%]",
-                    "on": "[%key:common::state::on%]",
-                    "paused": "[%key:common::state::paused%]",
+                    "idle": "Jõude",
```
Comment (Thomas55555) [inline custom_components/husqvarna_automower/translations/et.json:228]: Aren't the common strings working?

Comment (priiduonu) [reply custom_components/husqvarna_automower/translations/et.json:228]: They are. I decided to follow the style in `en.json`.

**Comment by Thomas55555** [comment]
I think there was something wrong in this commit. Can you switch back to the common keys?

File: custom_components/husqvarna_automower/translations/et.json (hunk 1 above)
Comment (Thomas55555) [reply custom_components/husqvarna_automower/translations/et.json:228]: I think there was something wrong in this commit. https://github.com/Thomas55555/husqvarna_automower/commit/eca2a390a015b37f8f1ddb14e55862da4db7b9d2
Can you switch back to the common keys?

**Comment by Thomas55555** [comment]
Thx @priiduonu

**Pull request merged**